service/service.py

```diff
@@ -230,6 +230,46 @@ def get(self):
 
         return response_content, status.HTTP_200_OK
 
+######################################################################
+#  PATH: /wishlists/{wishlist_id}
+######################################################################
+@api.route('/wishlists/<wishlist_id>')
+@api.param('wishlist_id', 'The Wishlist identifier')
+class WishlistResource(Resource):
+    ######################################################################
+    # RENAME WISHLIST
+    ######################################################################
+    @api.doc('rename_wishlist')
+    @api.response(404, 'No wishlist found.')
+    @api.response(400, 'Validation errors: "Invalid request: missing name"')
+    @api.expect(wishlist_model)
```

Comment:
I think this should be `@api.expect(create_wishlist_model)`. The only difference between `wishlist_model` and `create_wishlist_model` is the `id`. I think we can rename `create_wishlist_model` to something else to fit the update use case also.

Rest LGTM